parquet/src/file/metadata/reader.rs

```diff
@@ -118,27 +139,49 @@ impl ParquetMetaDataReader {
     }
 
     /// Enable or disable reading the page index structures described in
-    /// "[Parquet page index]: Layout to Support Page Skipping". Equivalent to:
-    /// `self.with_column_indexes(val).with_offset_indexes(val)`
+    /// "[Parquet page index]: Layout to Support Page Skipping".
     ///
     /// [Parquet page index]: https://github.com/apache/parquet-format/blob/master/PageIndex.md
+    #[deprecated(since = "56.0.0", note = "Use `with_page_index_policy` instead")]
```

Comment:
```suggestion
    #[deprecated(since = "56.0.1", note = "Use `with_page_index_policy` instead")]
```
These changes will go into 56.0.1

Reply:
I think technically it will be `56.1.0` (not `56.0.1`)

Reply:
D'oh 😅